chain-signatures/contract/src/lib.rs

```diff
@@ -373,6 +374,19 @@ impl VersionedMpcContract {
         }
     }
 
+    /// This is the derived key given path
+    pub fn my_derived_public_key(&self, path: String) -> PublicKey {
```

Comment:
Don't think `my` adds any more info here, and we should just keep consistent with verbs in contract interfaces. How about we just rename to `derive_public_key`

Reply:
changed it to derived_public_key() to be consistent with public_key()